src/main/java/io/cosmosoftware/kite/instrumentation/NetworkProfile.java

```diff
@@ -203,21 +188,25 @@ public String getCleanUpCommand() {
   public void setCleanUpCommand(String cleanUpCommand) {
     this.cleanUpCommand = cleanUpCommand;
   }
-
+  
   @Transient
-  public String getNit() {
-    return nit;
+  public String getCommand(List<String> nit) throws KiteTestException {
```

Comment:
NWInstrumentation only done on the first interface

Reply:
yes, defaultCommand is only on one interface.

but the custom command can be on several interfaces (in a single command that is).
the parameter is used just to replace %nit0 by nit.get(0)